oleesir/store-manager
# feat(add-products):implement add-product-template

#### What does this PR do?
implement add-products-template
#### Description of Task to be completed?
implement add-products-template
#### How should this be manually tested?
* Navigate to the UI folder and click on add-products.html in your browser
* You will see a add-products template
#### What are the relevant pivotal tracker stories?
[#161193844](https://www.pivotaltracker.com/story/show/161193844)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="screen shot 2018-10-12 at 10 30 47 pm" src="https://user-images.githubusercontent.com/17679511/46895029-8a7d6c00-ce6e-11e8-9e22-4bf2983efdd2.png">
